middleware/camera/onvif_zeep_camera_controller.py

```diff
@@ -0,0 +1,144 @@
+import logging
+from django.conf import settings
+from onvif import ONVIFCamera, ONVIFError
+from middleware.camera.abstract_camera import AbstractCameraController
+from middleware.camera.exceptions import InvalidCameraCrendentialsException
+from middleware.camera.types import CameraAsset
+from middleware.camera.utils import wait_for_movement_completion
+
+logger = logging.getLogger(__name__)
+
+
+# my_cam=ONVIFCamera("192.168.68.116", 80, "remote_user","2jCkrCRSeahzKEU", "/home/teleicu/.local/share/virtualenvs/teleicu_middleware_backend-pwPF-mjP/lib/python3.12/site-packages/wsdl/")
+class OnvifZeepCameraController(AbstractCameraController):
+
+    def __init__(self, req: CameraAsset) -> None:
+        try:
+            # cam = ONVIFCamera(
+            #     req.hostname, req.port, req.username, req.password, settings.WSDL_PATH
+            # )
+            cam = ONVIFCamera(req.hostname, req.port, req.username, req.password)
+        except ONVIFError as err:
+            logger.info(
+                "Exception raised while connecting to Camera with req: %s and reason: %s",
+                req,
+                err.reason,
+            )
+
+            raise InvalidCameraCrendentialsException
```

Comment:
make sure its actually camera auth error and not due to camera being offline

Reply:
we do not get a specific exception when this happens but we can verify it with the error message so for creds this is the error msg it throws 
The action requested requires authorization and the sender is not authorized
do you want me to throw this exception if this string is present in the error msg?

Reply:
in that case, let it be this way